AudioDeviceExample/ViewController.swift

```diff
@@ -202,6 +191,30 @@ class ViewController: UIViewController {
         self.dismissKeyboard()
     }
 
+    // MARK:- IBActions
+    @IBAction func connect(sender: AnyObject) {
+        connectButton.isEnabled = false
+        // Configure access token either from server or manually.
+        // If the default wasn't changed, try fetching from server.
+        if (accessToken == "TWILIO_ACCESS_TOKEN") {
+            TokenUtils.fetchToken(from: tokenUrl) { [weak self]
+                (token, error) in
+                DispatchQueue.main.async {
+                    if let error = error {
+                        let message = "Failed to fetch access token:" + error.localizedDescription
+                        self?.logMessage(messageText: message)
+                        self?.connectButton.isEnabled = true
+                        return
+                    }
+                self?.accessToken = token;
+                self?.connectToARoom()
```

Comment:
Align with the `if let`:

```suggestion
                    self?.accessToken = token;
                    self?.connectToARoom()
```